.github/ISSUE_TEMPLATE/desktop_window.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Window styling/size realted issue
+about: I want to apply custom styles to application window
+title: ""
+labels: ""
+assignees: ""
+---
+
+<!--
+Flutter itself has nothing to do with OS sepcific window styling
+You need to tune everything related to window appearance in macos/windows/linux projects
```

Comment:
This isn't entirely true. I would rather get these issues, and decide which ones should be feature requests for the API surface.